model/src/main/kotlin/config/Resolutions.kt

```diff
@@ -40,14 +41,22 @@ data class Resolutions(
      */
     @JsonAlias("evaluator_errors")
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    val ruleViolations: List<RuleViolationResolution> = emptyList()
+    val ruleViolations: List<RuleViolationResolution> = emptyList(),
+
+    /**
+     * Resolutions for vulnerabilities provided by the advisor.
+     */
+    @JsonAlias("advisor_errors")
```

Comment:
Minor point: The @JsonAlias is typically only necessary for reasons of backwards compatibility if a property has been renamed. As `vulnerabilities` is a new property, it should not be needed here.
Sorry for not noticing before.